Flutter/pdf-viewer/sticky-notes.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
```

Comment:
Can these be API linked?

<img width="854" alt="image" src="https://github.com/user-attachments/assets/28098063-f989-4011-b6b0-fe250862cbfd">

Reply:
There is no API links is available for the values in enumeration.
https://pub.dev/documentation/syncfusion_flutter_pdfviewer/latest/pdfviewer/PdfAnnotationMode.html